data/operators/amenity/school.json

```diff
@@ -7313,7 +7313,7 @@
       }
     },
     {
-      "displayName": "Department of Education and Training (Victoria)",
+      "displayName": "Department of Education (Victoria)",
```

Comment:
![图片](https://user-images.githubusercontent.com/42690037/210072419-406e6570-e7a1-4cad-90ee-b67d9ad04e3e.png)

Sorry, but I haven't found this changes in their website, and I think you can add old name as a matchName

Reply:
Hey Laoshu,

The Department is changing its name as part of the Machinery of Government changes, and it will take effect on Jan 01 2023. https://vpsc.vic.gov.au/about-public-sector/machinery-of-government/

I figured if I submitted it a day early it would avoid problems when the changeover happens! :)

Reply:
@LaoshuBaby The Department has now updated the logo in its website, and the rest of the site will be progressively changed as the transition to the new name continues. (Similar to the other departments in the State)

Would it be possible to progress this change? I’m planning on doing a bulk edit and I don’t want to introduce the change and have iD suggest it needs to be changed back. :)